config/routes/api.rb

```diff
@@ -13,6 +13,7 @@
 
     resources :encryptables, except: [:new, :edit] do
       resources :file_entries, only: [:create, :index, :destroy, :show]
+      resources :logs
```

Comment:
```suggestion
      resources :logs, only: [:index]
```
do we need :show too?